src/components/FormFieldset/FormFieldset.tsx

```diff
@@ -4,21 +4,30 @@ import cx from 'classnames';
 type FormFieldProps = {
   children: JSX.Element[] | JSX.Element;
   className?: string;
+  isDisabled?: boolean;
   legend: string;
+  legendClassName?: string;
 };
 
 export const FormFieldset = ({
   children,
   className,
-  legend
+  isDisabled,
+  legend,
+  legendClassName
 }: FormFieldProps) => {
   const classNames = cx('cc-form-fieldset', {
```

Comment:
There has been a recent move to having `classNames` as an object for readability, if there are multiple uses made of "classname" util, so this would become something along the lines of ...

```
const classNames = {
  fieldset: cx('cc-form-fieldset', {
    [className]: className
  }),
  legend: cx('cc-form-legend', {
    'is-disabled': isDisabled,
    [legendClassName]: legendClassName
  })
};
```

Reply:
updated :)